pool.go

```diff
@@ -142,10 +140,6 @@ func (p *Pool) replace(c *client) {
 }
 
 func (p *Pool) inc() bool {
-	if p.created >= p.max {
```

Comment:
Makes a lot of sense, since it is duplicated.

Reply:
Is this safe to merge?